src/hooks/useGetNodesCategoryCount.ts

```diff
@@ -0,0 +1,64 @@
+import { useSelector } from 'react-redux';
+
+import {
+  stakeSelector,
+  stakeExtraSelector,
+  nodesOverviewSelector
+} from 'redux/selectors';
+import { NodeStatusEnum, NodeTypeEnum } from 'types';
+
+export const useGetNodesCategoryCount = ({
+  showGlobalValues
+}: {
+  showGlobalValues?: boolean;
+}) => {
+  const {
+    unprocessed: { queueSize, auctionValidators, totalObservers }
+  } = useSelector(stakeSelector);
+  const {
+    unprocessed: { totalNodes, totalValidatorNodes }
+  } = useSelector(stakeExtraSelector);
+  const { nodes, isFetched: isNodesOverviewFetched } = useSelector(
+    nodesOverviewSelector
+  );
+
+  if (showGlobalValues) {
+    return {
+      totalNodes,
+      totalValidatorNodes,
+      auctionValidators,
+      queueSize,
+      totalObservers
+    };
+  }
+
+  if (isNodesOverviewFetched && nodes.length > 0) {
+    const overviewTotal = nodes.length;
+    const overviewValidatorNodes = nodes.filter(
```

Comment:
not sure about the no of nodes, but I think it could improve a bit if these filters would be memoized

Reply:
as the number of nodes isn't too high ( mostly @30-40, max 150 ) I think that for now memoization would add a bit of unnecessary overhead. Will keep this in mind for larger arrays